pom.xml

```diff
@@ -425,7 +425,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>pipeline-build-step</artifactId>
-            <version>2.5.1</version>
+            <version>2.7-SNAPSHOT</version>
```

Comment:
@sophistifunk Can we deploy SNAPSHOT version of pipeline-build-step and use the version number deployed there so that CI build passes?